lib/Dialect/FIRRTL/Import/FIRAnnotations.cpp

```diff
@@ -864,6 +864,7 @@ bool circt::firrtl::scatterCustomAnnotations(
         NamedAttrList foo;
         foo.append("class", dict.get("class"));
         foo.append("id", id);
+        foo.append("word", IntegerAttr::get(IntegerType::get(context, 64), i));
```

Comment:
Nice! Clearly this was missing in hindsight... 😄

Reply:
:smile: Somehow there was no need for that before. Or rather, the pass just started enumerating the memory words as it went along. But when fixing the other stuff I realized that this would essentially discard the port order specified in the `taps` field of the `MemTapAnnotation`.